pkg/test/framework/core/settings.go

```diff
@@ -82,11 +90,11 @@ func DefaultSettings() *Settings {
 func (s *Settings) String() string {
 	result := ""
 
-	result += fmt.Sprintf("Environment: %v\n", s.Environment)
-	result += fmt.Sprintf("TestID:      %s\n", s.TestID)
-	result += fmt.Sprintf("RunID:       %s\n", s.RunID.String())
-	result += fmt.Sprintf("NoCleanup:   %v\n", s.NoCleanup)
-	result += fmt.Sprintf("BaseDir:     %s\n", s.BaseDir)
-
+	result += fmt.Sprintf("Environment:  %v\n", s.Environment)
+	result += fmt.Sprintf("TestID:       %s\n", s.TestID)
+	result += fmt.Sprintf("RunID:        %s\n", s.RunID.String())
+	result += fmt.Sprintf("NoCleanup:    %v\n", s.NoCleanup)
+	result += fmt.Sprintf("BaseDir:      %s\n", s.BaseDir)
+	result += fmt.Sprintf("Selector:       %v\n", s.Selector)
```

Comment:
Maybe it's just the code review tool, but the value doesn't seem to line up with the others above.